cli/dcoscli/job/main.py

```diff
@@ -575,6 +575,9 @@ def _add_job(job_file):
     if 'id' not in full_json:
         raise DCOSException("Jobs JSON requires an ID.")
 
+    if 'disk' not in full_json['run']:
+        full_json['run']['disk'] = 0
```

Comment:
run is required as part of the schema and there is no such thing as a job with a run command.  If you want me to provide the check I will.  thoughts?